chaoscenter/web/src/views/AccountSettingsUserManagement/AccountSettingsUserManagement.tsx

```diff
@@ -230,16 +244,35 @@ export default function AccountSettingsUserManagementView(
           small
           loading={useUsersQueryLoading}
           noData={{
-            when: () => !usersData?.length,
+            when: () => !currentItems?.length,
```

Comment:
I believe this will still be `!usersData?.length` as the pagination is solely frontend based, hence it will not generate page index that ever go out of bounds, so using `!currentItems?.length` is not necessary here.